src/Networking/NexusMods.Networking.Downloaders/Interfaces/IDownloadService.cs

```diff
@@ -0,0 +1,95 @@
+using NexusMods.DataModel.Loadouts;
+using NexusMods.DataModel.RateLimiting;
+using NexusMods.Networking.NexusWebApi.Types;
+using NexusMods.Paths;
+
+namespace NexusMods.Networking.Downloaders.Interfaces;
+
+/// <summary>
+/// This is a 'download manager' of sorts.
+/// This service contains all of the downloads which have begun, or have already started.
+/// </summary>
+public interface IDownloadService
+{
+    /// <summary>
+    /// Contains all downloads managed by the application.
+    /// </summary>
+    List<IDownloadTask> Downloads { get; }
```

Comment:
This should be a ReadOnlyObservableCollection, so that we can see the small delta changes, and not have to deal with large lists of items once the number of downloads reach into the thousands.